### PR TITLE
Make chart generation more readable

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Review by pradh — APPROVED** [review]
Thanks for the improvement!

**Review by beets — APPROVED** [review]

File: server/lib/nl_page_config.py [hunk 1]
```diff
@@ -56,61 +56,53 @@ def get_sv_name(svs):
...
+    stat_var_spec_map[sv_key] = subject_page_pb2.StatVarSpec(stat_var=sv,
+                                                             name=sv2name[sv])
+  block.columns[0].tiles.append(tile)
   # Line chart for the stat var per capita
-  tile = column.tiles.add()
-  tile.type = subject_page_pb2.Tile.TileType.LINE
-  tile.title = "Per Capita"
+  tile = subject_page_pb2.Tile(type=subject_page_pb2.Tile.TileType.LINE,
+                               title="Per Capita",
+                               stat_var_key=[])
```
Comment (pradh) [inline server/lib/nl_page_config.py:98]: stat_var_key=[] needed?

Comment (shifucun) [reply server/lib/nl_page_config.py]: Not really, removed

**Comment by shifucun** [comment]
thanks!

**Pull request merged**